client/src/config.js

```diff
@@ -1,9 +1,9 @@
 const config = {
   // OAuth info (copied from the FusionAuth admin panel)
-  clientID: '48f671cc-1627-41b5-aed1-260b80a9f742',
-  clientSecret: 'pqQj3uOj7q7BLe0ImvgSxCvKUTuczAth1vjAmkntve8',
+  clientID: '85a03867-dccf-4882-adde-1a79aeec50df',
```

Comment:
If it doesn't break anything in this code, our code style guide would name this `clientId` - and `applicationId` respectively.

Reply:
Will be fixed in next checkin.